plugins/a11ychecker.md

```diff
@@ -161,91 +295,212 @@ If `a11ychecker_allow_decorative_images` is set to `false`, the Accessibility Ch
 
-#### Example: a11ychecker_html_version
+#### Example: Using `a11ychecker_html_version`
 
 ```js
 tinymce.init({
-    selector: 'textarea',
-    plugins: 'a11ychecker',
-    toolbar: 'a11ycheck',
-    a11ychecker_html_version: 'html5'
+  selector: 'textarea',
+  plugins: 'a11ychecker',
+  toolbar: 'a11ycheck',
+  a11ychecker_html_version: 'html5'
+});
+```
+
+### `a11ychecker_ignored_rules`
+
+The `a11ychecker_ignored_rules` option prevents specific Accessibility Checker rules being checked. This feature allows developers to skip rules that flag known issues in content until the issue is fixed.
+
+**Type:** `String`
+
+**Default Value:** `''`
+
+**Possible values:** `'D1'`, `'D2'`, `'D3'`, `'D4O'`, `'D4U'`, `'D5A'`, `'D5B'`, `'D5C'`, `'H93'`, `'I1'`, `'I2'`, `'T1'`, `'T2'`, `'T3'`, `'T4A'`, `'T4B'`, `'T4C'`
+
+#### Example: Using `a11ychecker_ignored_rules`
+
+This examples shows how to ignore the following checks (rules):
+
+* D2 - Sequential headings
+* I2 - Alt text filename
+* T4B - Table headers
+
+```js
+tinymce.init({
+  selector: 'textarea',  // change this value according to your html
+  plugins: 'a11ychecker',
+  toolbar: 'a11ycheck',
+  a11ychecker_ignored_rules: 'D2, I2, T4B'
+});
+```
+
+### `a11ychecker_issue_url_callback`
+
+The `a11ychecker_issue_url_callback` option is used to change the target URL for the "Click for more info" button (![help icon - a question mark inside a circle]({{site.baseurl}}/images/icons/help.svg)) in the Accessibility Checker dialog. By default, the "more info" links will point to `{{site.url}}/docs/plugins/a11ychecker/#<ruleId>`, such as `{{site.url}}/docs/plugins/a11ychecker/#D1`. This option can be used to set the target URL to a page or pages outside {{site.url}}/docs/.
+
+**Type:** `Function`
+
+**Default Value:** `undefined`
+
+#### Example: Using `a11ychecker_issue_url_callback`
+
+This example shows how to change the link for the "Click for more info" button (![help icon - a question mark inside a circle]({{site.baseurl}}/images/icons/help.svg)) on the Accessibility Checker dialog to point to anchors at `www.example.com/tinymce/a11ychecker/more_info`.
+
+```js
+tinymce.init({
+  selector: 'textarea',  // change this value according to your html
+  plugins: 'a11ychecker',
+  toolbar: 'a11ycheck',
+  a11ychecker_issue_url_callback: function (ruleId) {
+    return `https://www.example.com/tinymce/a11ychecker/more_info#${ruleId}`
+  };
 });
 ```
 
 ### `a11ychecker_level`
 
 This configuration option sets the [WCAG level](https://www.w3.org/TR/WCAG20/#conformance) to use when checking issues.
 
-For example, the "Text must have a contrast ratio of at least ..." rule when using **AA** will trigger when the contrast ratio is less than 4.5:1, however when using **AAA** the rule will trigger when the ratio is less than 7.0:1.
+For example, the "Text must have a contrast ratio of at least ..." rule:
+
+* When using **AA**, Accessibility Checker will check for a contrast ratio of less than 4.5:1.
```

Comment:
To me, "check for a contrast ratio of $X" would read as the plugin trying to verify that the contrast ratio _is_ $X, and erroring if it is _not_ $X. I might be wrong though, feel free to ignore

Reply:
fair, I'll reword

Reply:
Fixed by a366c514e3559853e05910bd21715ecf47024b8a